### DISCUSSION TITLE
I need help (I don't know how to deactivate marlin mode)

### DISCUSSION THREAD

**Discussion by benjaminastetego** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2809] [General]
Hi, I have an artilllery genius pro, I installed the touch software for tft but one day I saw the option to switch to marlin mode and out of curiosity I pressed it and now it stayed in that picture and I do not know how to return, I would use a sd card to make a file with reset.exe, but I have no sd card now, is there a way to get it out of there?
![IMG_4526](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/137271278/80def921-ad49-4010-9dae-add751a74e97)

**Comment by kisslorand** [comment]
Make a long press on the TFT (3 sec).

**Reply by IIIMartix** [reply]
I have same problem. It's does'nt help for me, I tryed 6 and more seconds, in a different places on touch many times — nothing ;(

**Comment by kisslorand** [comment]
Put an empty "reset.txt" file on an SDCard and boot the TFT with that SDCard inserted into the TFT's SDCard slot. 

https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware#hard-reset
